### PR TITLE
Adding Instabug to issue tracking section

### PR DESCRIPTION
**Pull request by yhelmy1**

### CODE REVIEW

**Comment by yhelmy1** [comment]
@ripienaar Here is the PR! Thank you so much :)

File: README.md [hunk 1]
```diff
@@ -486,6 +486,7 @@ Table of Contents
    * [plan.io](https://plan.io/) — Project Management with Repository Hosting and mor options. Free for 2 users with 10 customers and 500MB Storage
    * [taskade.com](https://taskade.com/) — Real-time collaborative task lists and outlines for teams
    * [zenkit.com](https://zenkit.com) - Project management and collaboration tool. Free for up to 5 members, 5 GB attachments.
+   * [Instabug](https://instabug.com) —  A comprehensive bug reporting and in-app feedback SDK that is helping companies worldwide build better apps.
```
Comment (ripienaar) [inline README.md]: good good, please mention the details of the free plan and it's good to go 👍

**Comment by yhelmy1** [comment]
@ripienaar I added the free plan edits. Thanks again :)

**Comment by ripienaar** [comment]
thank you :)

**Pull request merged**